pallets/liquidity-pools/src/tests/inbound.rs

```diff
@@ -773,3 +822,433 @@ mod handle_cancel_redeem_order {
 		}
 	}
 }
+
+mod freeze {
```

Comment:
I think these tests are checking the extrinsics, not the methods found in `inbound.rs`. Could they be in `tests.rs` instead?

Reply:
Yes sorry, will refactor.

Reply:
[a5f2e6e](https://github.com/centrifuge/centrifuge-chain/pull/1947/commits/a5f2e6e272c2b8400190994a4529f97ca012ad46)